### ISSUE TITLE
Cannot GET /

### ISSUE THREAD

**Issue by themicster** [justinleewells/pogo-optimizer#78]
I keep getting errors from port 3000 that says cannot get /, I did see it once on my phone, it seemed to auto reload after I had logged into pogo then switched back to chrome. But then when I tried to reload I get this Cannot GET / error now all the time.

**Comment by justinleewells** [comment]
Follow the newest version of instructions if you just pulled. There's a whole new build system.

**Comment by themicster** [comment]
When I built the docker file myself, I can get the spreadsheet page to show on my phone, but not on my laptop. The page shows up but with no data. But I'm no longer getting that GET error, so that is good.

**Comment by justinleewells** [comment]
Are you going to localhost:3000 on your laptop?

**Comment by themicster** [comment]
If I load it from a different IP address than the one my phone using via the wifi, will it not show me any data? Well I tried connecting through the same IP as my phone, but I'm still not seeing anything from my desktop browser. I've tried with IE and Chrome. It's actually running on one of my servers in a datacenter. So I'm not going to localhost, I'm going to http://my.ip:3000
It works from my phone, I wonder if its because it's going through the proxy?

**Comment by themicster** [comment]
I'm happy with this even if I can only use it from my phone. It's pretty cool. I can see my only 100% IV is the snorlax I caught last night.

**Comment by justinleewells** [comment]
Yeah, the datacenter setup definitely complicates things. I'm not sure how you'd fix it.

Congrats on the 100% Snorlax! He's a top tier defender.

**Comment by Ardakilic** [comment]
I'm also getting the same error:

```
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::3000
  at Object.exports._errnoException (util.js:1008:11)
  at exports._exceptionWithHostPort (util.js:1031:20)
  at Server._listen2 (net.js:1253:14)
  at listen (net.js:1289:10)
  at Server.listen (net.js:1385:5)
  at EventEmitter.listen (/root/pogo-optimizer/node_modules/express/lib/application.js:617:24)
  at Object.<anonymous> (/root/pogo-optimizer/index.js:98:5)
  at Module._compile (module.js:541:32)
  at Object.Module._extensions..js (module.js:550:10)
  at Module.load (module.js:458:32)
  at tryModuleLoad (module.js:417:12)
  at Function.Module._load (module.js:409:3)
  at Module.runMain (module.js:575:10)
  at run (bootstrap_node.js:352:7)
  at startup (bootstrap_node.js:144:9)
  at bootstrap_node.js:467:3
```

**Comment by justinleewells** [comment]
That means you already have something running on port 3000.

**Comment by Ardakilic** [comment]
@justinleewells Yup, i can confirm it works now. It was a zombie process. Thanks! :+1:

**Comment by justinleewells** [comment]
No problem! Glad I could help.

**Comment by paanvaannd** [comment]
I'm running into the exact same problem and I've built using Docker as well.

I've never tinkered with Docker before this. I ran this program smoothly 2 days ago and now can't get it to work. I just quit Docker (running on a Mac) without specifically stopping processes. Is that an issue? What command do I issue Docker to stop pogo-optimizer? I thought it would be something along the lines of 'docker stop pogo-optimizer'  but that doesn't work... and what should I do to kill processes running on port 3000 to make sure it's clear for use by pogo-optimizer?
